src/main/helpers.ts

```diff
@@ -27,3 +29,12 @@ export const calculateOriginalPrice = (
 
   return formattedAmount;
 };
+
+export const transformCategoriesToOptions = (
+  categories: Category[],
+): { value: string; label: string }[] => {
+  return categories.map((category) => ({
```

Comment:
`{ value: string; label: string }` type for it please. It will be reusable

Reply:
@VYaniv , it will be realise in the task 42